File: scripts/bench/bench.cpp
#include <iostream>
#include <cstdlib>
using namespace std;

int main() {
    int N = 5000000;
    const char* env = getenv("ECO_BENCH_N");
    if (env) N = atoi(env);
    long long s = 0;
    for (int i = 0; i < N; i++) s += (i & 1);
    cout << s << "\n";                 // optional correctness output
    cout << "ECO_OPS: " << N << "\n"; // required
    return 0;
}

File: scripts/bench/Bench.java
public class Bench {
  public static void main(String[] args) {
    int N = Integer.parseInt(System.getenv().getOrDefault("ECO_BENCH_N", "5000000"));
    long s = 0;
    for (int i = 0; i < N; i++) {
      s += (i & 1);
    }
    System.out.println(s);                 // optional correctness output
    System.out.println("ECO_OPS: " + N);   // required
  }
}
